source/2019-07-26-the-ember-times-issue-108.md

```diff
@@ -30,13 +32,19 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## [Sentry Error Reporting for Ember.js 📈](https://simplabs.com/blog/2019/07/15/sentry-and-ember/)
+
+[Sentry](https://sentry.io/for/ember/) is an open-source error tracking tool that helps you monitor and fix production errors in your Ember app.
+
```

Comment:
I would include something about why it was written --> Something about 
> Well... it was the official browser JavaScript client for Sentry. In 2018 it has been replaced by @sentry/browser with an entirely new API

Reply:
Yep, I think it's a great idea to emphasize the history here. I'll make the change, thanks.